python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -517,17 +517,22 @@ def do_evaluate(
         elif typ == "parquet":
             parquet_options = config_options.get("parquet_options", {})
             if parquet_options.get("chunked", True):
+                options = plc.io.parquet.ParquetReaderOptions.builder(
+                    plc.io.SourceInfo(paths)
+                ).build()
+                # We handle skip_rows != 0 by reading from the
+                # up to n_rows + skip_rows and slicing off the
+                # first skip_rows entries.
+                # TODO: Remove this workaround once
+                # https://github.com/rapidsai/cudf/issues/16186
+                # is fixed
+                nrows = n_rows + skip_rows
+                if nrows != -1:
```

Comment:
Silly question:  Shouldn't this be a less permissive check?
```suggestion
                if nrows > -1:
```

Reply:
This might hold for the other spots in the code here.

Reply:
Yeah, will fix